shared/docs/flows/README.md

```diff
@@ -0,0 +1,3 @@
+# Flows
```

Comment:
I think this is a great addition and we can put in here more flows as well. We just need to keep this stuff updated.

On the latter, not sure if we should enforce this via PR reviews or if we should add to the PR template. 
The current wording *could* include this already but a new contributor wouldn't know where to look at.

```markdown
- [ ] If applicable, I have added new diagrams using [mermaid.js](https://mermaid-js.github.io)
```

Reply:
Short-term solution: https://github.com/pokt-network/pocket/commit/245cb6e5481bed56bcbc2c5b1a0a2e9d0675314c

Long-term solution: https://github.com/pokt-network/pocket/issues/255